src/Entity/MajorVersion.php

```diff
@@ -282,33 +272,40 @@ public function toArray(): array
 
     public function isActive(): bool
     {
-        $dateTime = new \DateTimeImmutable();
-        return null === $this->getMaintainedUntil()
-            || $dateTime <= $this->getMaintainedUntil();
+        $dateTimeImmutable = new \DateTimeImmutable();
+        if (null === $this->getMaintainedUntil()) {
+            return true;
+        }
+        return $dateTimeImmutable <= $this->getMaintainedUntil();
     }
 
     public function isElts(): bool
     {
-        $dateTime = new \DateTimeImmutable();
-        return $this->getMaintainedUntil() != null
-            && $this->getEltsUntil() != null
-            && $dateTime > $this->getMaintainedUntil()
-            && $dateTime <= $this->getEltsUntil();
+        $dateTimeImmutable = new \DateTimeImmutable();
+        if ($this->getMaintainedUntil() == null) {
+            return false;
+        }
+        if ($this->getEltsUntil() == null) {
+            return false;
+        }
+        if ($dateTimeImmutable <= $this->getMaintainedUntil()) {
+            return false;
+        }
+        return $dateTimeImmutable <= $this->getEltsUntil();
     }
 
     /**
-     * Specify data which should be serialized to JSON
-     *
-     * @link http://php.net/manual/en/jsonserializable.jsonserialize.php
-     * @return mixed data which can be serialized by <b>json_encode</b>,
-     * which is a value of any type other than a resource.
-     * @since 5.4.0
-     */
+    * Specify data which should be serialized to JSON
+    *
+    * @link http://php.net/manual/en/jsonserializable.jsonserialize.php
+     * @return array<string, mixed> data which can be serialized by <b>json_encode</b>, which is a value of any type other than a resource.
+    * @since 5.4.0
+    */
```

Comment:
Is repeating the PHP docs necessary?

Reply:
Absolutely not. I guess Susi copy&pasted it. Will remove as much as possible...

Reply:
done